### PR TITLE
Update upgrades.md

### PR DESCRIPTION
**Pull request by kevinayres**
Add consideration for the current 'Rancher Setup' AWS Marketplace offering, which uses the deployment name of 'rancher-setup'.

### CODE REVIEW

File: docs/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades.md [hunk 1]
```diff
@@ -131,6 +131,17 @@ There will be more values that are listed with this command. This is just an exa
 
 :::
 
+:::tip 
+
+Your deployment name may vary; for example, with the AWS Marketplace offering of Rancher Setup, the deployment name is 'rancher-stable'.
```
Comment (martyav) [inline docs/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades.md:136]:
```suggestion
Your deployment name may vary; for example, if you're deploying Rancher through the AWS Marketplace, the deployment name is 'rancher-stable'. 
```

**Review by martyav — CHANGES_REQUESTED** [review]

**Comment by kevinayres** [comment]
It looks like this was resolved but not published yet.  The change looks good - thank you.

**Review by martyav — APPROVED** [review]

**Comment by martyav** [comment]
@kevinayres before we merge, do you know if the PR changes also relate to v2.6 or earlier?

**Comment by martyav** [comment]
I decided to merge anyway, since the information probably pertains more heavily to 2.7 than to earlier versions, as it is the latest

**Comment by kevinayres** [comment]
Apologies - the current 'Rancher Setup' AWS Marketplace offering is 2.6. It will be replaced with a new MP listing of 2.7.5> and hopefully have the correct deployment name. We'll update both the rancher docs and the Getting Started Guide again after the new listing comes out and we find out what deployment name is uses.

**Comment by martyav** [comment]
@kevinayres No problem. I'll just sync the relevant 2.6 pages

**Pull request merged**